overview/demo/shell.nix

```diff
@@ -59,6 +83,15 @@ in
             };
             default = { };
           };
+          usage-instructions = mkOption {
+            type = lines;
```

Comment:
What if we make this a function that takes string and formats it to print the `Demo Usage Instructions` block?